suzieq/engines/pandas/ospf.py

```diff
@@ -425,12 +425,15 @@ def _get_peernames(self, df: pd.DataFrame, columns: List[str],
         if newdf.empty:
             return df
 
-        lldp_df = self._get_table_sqobj('lldp').get(
+        addr_df = self._get_table_sqobj('address').get(
             namespace=kwargs.get('namespace', []),
-            hostname=kwargs.get('hostname', []),
-            use_bond='True')
+            hostname=kwargs.get('hostname', []))
+        if not addr_df.empty:
+            addr_df = addr_df.query('ipAddressList.str.len() != 0')
```

Comment:
Should we be using `ipAddress` (v4 and v6) instead of `ipAddressList` (v4 only)? I'm not sure anyone uses ipv6 for ospf but it's a low effort to support them both.